cranelift/Cargo.toml

```diff
@@ -13,13 +13,21 @@ rust-version.workspace = true
 [lints]
 workspace = true
 
+[package.metadata.docs.rs]
+# Ask Cargo to build docs with `--all-features` set to true
+all-features = true
```

Comment:
You need to add this to `cranelift/codegen/Cargo.toml`, not `cranelift/Cargo.toml`. `cranelift/Cargo.toml` is for the cranelift-tools crate which isn't published to crates.io anyway. Also instead of enabling all non-default features, I would suggest only enabling `all-arch`. Some like `souper-harvest` and `disas` don't affect the public api while still making building the documentation slower.

Reply:
- Updated `cranelift/codegen/Cargo.toml` to enable the `all-arch` feature:
  ```toml
  [package.metadata.docs.rs]
  features = ["all-arch"]
  ```
- Verified feature requirements are displayed in the docs using:
  ```bash
  RUSTDOCFLAGS="--cfg docsrs" cargo +nightly doc --open
  ```